firmware/quadruna/BMS/src/io/io_ltc6813.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+#include <stdbool.h>
+#include <stdint.h>
+
+// Global Variables
+#define NUM_SEGMENTS 4
```

Comment:
I would argue these defines deserve they’re own header file, something along the lines of
`packConfig.h`